ManatiCR/manatime
# 17: Timesheet submit.

**Steps to test**
- Create some nodes: project, task, timesheet, timeentries, Client.
- Check if the button 'Submit timesheet' run successfully.

**DoD**
- Submit button (form?)
- Set timesheet status to published
- Set time entries to Submitted.
- If submit is done in the middle of a week (ex. wednesday 31 of a given month), create a new timesheet for the rest of the week.

[issue link](https://github.com/ManatiCR/manatime/issues/17)
